pkg/controllers/management/eksupstreamrefresh/eks_upstream_refresh.go

```diff
@@ -182,10 +215,27 @@ func GetComparableUpstreamSpec(secretsCache wranglerv1.SecretCache, cluster *mgm
 		}
 
 		nodeGroupStates = append(nodeGroupStates, ng)
+		var nodeGroupMustBeDeleted string
+		if len(ng.Nodegroup.Health.Issues) != 0 {
+			var issueMessages []string
+			for _, issue := range ng.Nodegroup.Health.Issues {
+				issueMessages = append(issueMessages, aws.StringValue(issue.Message))
+				if !controller.NodeGroupIssueIsUpdatable(aws.StringValue(issue.Code)) {
```

Comment:
The template missing is the only one that totally stops syncing from working right? I'm wondering now if we should continue treating them all the same since the eks handler is going to essentially shut down until it's resolved.